### PR TITLE
Release 1

### PR DESCRIPTION
**Pull request by cristianbgp**
These user stories were done:

- User must login in the App
- User must register in the App
- Owner user can create a club
- Owner user can create a sports field
- Owner user can see a list of their clubs
- Owner user can see a list of their sports fields grouped by club
- Owner user can see the options to create a club or a sports field from the “Club list” view

### CODE REVIEW

File: api/db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+puts "Init seed"
+User.destroy_all
+User.create(name: 'Lian Nivin', email: 'liam@kampu.pe', role: "regular", password: '123456')
+User.create(name: 'Cristian Berly', email: 'berli@kampu.pe', role: "owner", password: '123456')
+
+clubs = Club.create([{name: "Club #1", address: 'Jr cayumba 440',
+schedule: {
+  'monday-friday': {
```
Comment (liannivin) [inline api/db/seeds.rb]: What do you mean with `monday-friday`?

Comment (liannivin) [inline api/db/seeds.rb]: How do you count `monday, tuesday, wednesday, thrusday and friday` with that?

File: client/src/components/owner-create-button.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    setActive(!active);
+  }
+
+  return (
+    <div css={active ? styleListContainer : styleButtonContainer}>
+      <div css={active ? styleButtonsContainerActive : styleButtonsContainer}>
+        <Link to="/create-club" css={styleButton}>
+          Create Club
+        </Link>
+        <Link to="/create-sport-field" css={styleButton}>
```
Comment (liannivin) [inline client/src/components/owner-create-button.js]: Have you thought replacing that into nested links like `/create/sport-field`?

File: api/db/seeds.rb (hunk 1 above)
Comment (condef5) [reply api/db/seeds.rb]: Hey, I assume that the club will be open from Monday to Friday, so I use the key `monday-friday`.

File: client/src/components/owner-create-button.js (hunk 2 above)
Comment (condef5) [reply client/src/components/owner-create-button.js]: Of course, in the future, We will change this route by `/owner/sport-field/create`

**Pull request merged**